telemetry/definitions/commonDefinitions.json

```diff
@@ -30,11 +30,26 @@
             ],
             "description": "AWS filetype kind"
         },
+        {
+            "name": "cloudWatchLogsPresentation",
+            "allowedValues": ["ui", "text"],
+            "description": "Presentation mode used in a CloudWatch Logs operation"
+        },
+        {
+            "name": "cloudWatchResourceType",
+            "allowedValues": ["logGroup", "logGroups", "logStream"],
```

Comment:
Looks like we're treating a _collection_ of a resource as a concept in its own right. I've thought about this a lot and I think it's appropriate, however, we should think of a better name than just `logGroups`. Too easily confused with the singular `logGroup`. It may not be clear that we're refreshing the data provided by a `list` call rather than simply refreshing multiple log groups.

`logGroupCollection` is verbose though it may be more obvious? Maybe we don't make changes for this PR, but let's see if we can figure out a standard way to describe this idea.

Reply:
When I was reading the design doc to make this change, I initially thought it was a typo - "logGroups and logGroup?" - I like your suggestion to reduce confusion. Will update.